reactioncommerce/reaction-admin
# Fix navigation tree not showing up

Signed-off-by: Loan Laux <loan@outgrow.io>

Resolves #256 
Impact: **critical**
Type: **bugfix**

## Issue
The Navigation page wasn't working because of the navigation tree not showing up.

## Solution
Update `react-dnd` and `react-dnd-html5-backend` to 9.5.1, and move `DndProvider` up the component tree (at `<App />` level) to avoid conflict with `react-router`.

## Breaking changes
None.

## Testing
1. Visit the Navigation page.
2. See the navigation tree.
3. Try moving stuff around. It should work.
